blockchain/reactor_test.go

```diff
@@ -195,3 +196,4 @@ func (tp *bcrTestPeer) IsOutbound() bool                      { return false }
 func (tp *bcrTestPeer) IsPersistent() bool                    { return true }
 func (tp *bcrTestPeer) Get(s string) interface{}              { return s }
 func (tp *bcrTestPeer) Set(string, interface{})               {}
+func (tp *bcrTestPeer) QuitChan() <-chan struct{}             { return tp.Quit }
```

Comment:
Just add Quit() to BaseService.

Reply:
https://github.com/tendermint/tmlibs/pull/150